src/test/scala/edu/ie3/simona/agent/grid/ThermalGridIT.scala

```diff
@@ -586,16 +587,16 @@ class ThermalGridIT
       // point nor one of it reached any boundary.
       resultListener.expectNoMessage()
 
-      scheduler.expectMessage(Completion(heatPumpAgent, Some(31940)))
+      scheduler.expectMessage(Completion(heatPumpAgent, Some(31939)))
```

Comment:
Just to be sure: What's the cause of this?

Reply:
Same here, for better traceability we could move this in a separate PR upfront, if you like.